harness/determined/cli/job.py

```diff
@@ -100,6 +87,83 @@ def update(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def process_updates(args: Namespace) -> None:
+    session = setup_session(args)
+    for arg in args.operation:
+        inputs = {"session": session}
+        values = arg.split(".")
+        if len(values) != 2:
+            raise ValueError(
+                f"Invalid operation specified for job {values[0]}. "
+                f"Please ensure they are formatted as <jobID>.<operation>=<value>."
+            )
+        inputs["job_id"] = values[0]
+        operation = values[1].split("=")
+        if len(operation) != 2:
+            raise ValueError(
+                f"Operation for job {values[0]} has invalid format. "
+                f"Please ensure it is formatted as <jobID>.<operation>=<value>."
+            )
+        inputs[operation[0]] = operation[1]
```

Comment:
will it be nicer to parser this using a regexp?
i.e. `r'(?P<job_id>[0-9]+)\.(?P<operation>priority|weight|resource-pool|ahead-of|behind-of)=(?P<value>[\w]+)'`
(I have not tested this regexp)

Reply:
on 2nd thought, with regexp you won't be to provide "nice" error messages as easy.